### DISCUSSION TITLE
Testing mu-argus

### DISCUSSION THREAD

**Discussion by ppdewolf** [sdcTools/UserSupport#239] [Team Testers]
Could you all please leave a comment whether you are able to test version 5.1.7 of mu-argus?
Also let me know if you are _not_ able to test. That way I know what to expect.

**Comment by KarinaKelleher** [comment]
I will be able test this version (myself and my colleague Tim Linehan will be working on this), 
Thanks, Karina

**Comment by ppdewolf** [comment]
Thanks a lot Karina and Tim! Appreciated!

**Comment by julienjamme** [comment]
I'll do some tests by the end of July, most likely next week.
Julien

**Comment by ppdewolf** [comment]
Thanks Julien!

**Comment by ppdewolf** [comment]
For those of you who will test mu-argus 5.1.7:
I've just uploaded a [new build](https://github.com/sdcTools/muargus/releases/tag/v5.1.7b3) with some bug fixes, found by some testers (in the "MakeProtectedFile" window you could not select all suppression methods).

[Version 5.1.7build3](https://github.com/sdcTools/muargus/releases/tag/v5.1.7b3)

**Comment by julienjamme** [comment]
Peter-Paul,
Here are feedbacks on some tests I did on the new release of $\mu$-argus.  

### About TRS

- I compare the results of TRS with one similarity variable produced by the v5.1.6b2 and the v5.1.7b3: With the same seed and the same parameters obviously, the results are identical. For this comparison, I compare the two protected files with `identical()` R function - if we rename the columns of the previous release's dataset first.
- The fact that the protected files contains now the names of the original dataset is much appreciated.
- The use of multiple similarity variables seems to work well. 
- The file called 'NonSwappedHID.text' contains real IDs for HH (as far as I understand the result). It's easier to use than that of the previous release.

=> Some very useful improvements and no problem revealed during these tests.


### About 'Make Protected Files'

- The options on HID (keep, remove or sequence number) work as expected.
- Nevertheless, *the option **`write records in random number`** was always unavailable for me. I've never been able to choose it. I don't know if it's a normal behaviour.*

![image](https://user-images.githubusercontent.com/22718625/181788232-ec63b1fa-c3c6-43ff-932d-bd5dc07b4456.png)

**Comment by ppdewolf** [comment]
Thanks a lot Julien!

About the random ordering when saving the file: I don't know why, but it appears that this option is only available when the microdata is of fixed format (so no csv). I'll have to go deeper into the code to find out about a possible reason. Maybe you could add this as an issue to github? So it will be documented that way ;-)

**Comment by julienjamme** [comment]
Ok I can write an issue. But, as the repo on sdctools is a fork there is no "Issue" tab. Do you want me to write the issue on the original repo: https://github.com/ppdewolf/muargus even ?

**Comment by ppdewolf** [comment]
There should be an Issue tab on https://github.com/sdcTools/UserSupport

**Comment by ppdewolf** [comment]
Just added this issue myself on https://github.com/sdcTools/UserSupport/issues/188
